src/commands/mod.rs

```diff
@@ -1430,6 +1440,7 @@ pub struct GetInfoResult {
     pub vaults: usize,
     pub managers_threshold: usize,
     pub descriptors: GetInfoDescriptors,
+    pub participant_type: String,
```

Comment:
Thank you for the PR.

Please use [`UserRole`](https://github.com/revault/revaultd/blob/master/src/utils.rs#L28), you may need to implement `fmt::Display` to it and add ` #[serde(serialize_with = "ser_to_string", deserialize_with = "deser_from_str")]` to the `GetInfoResult` field

Reply:
Hey, I was looking through the codebase and I found that `UserRole` has been declared under `#[cfg(test)]` which means that it can't be used outside tests, I confirmed it by looking through the pre-existing uses of `UserRole` which have all been used in tests only.

Maybe I'm missing something but could you direct me towards a possible solution?

Reply:
You are right, I think it is better to move `UserRole` to revaultd.rs and outside of the #[cfg(test)].
Best case would be to have in this PR:
1. a commit to move UserRole to revaultd.rs and modify every tests using it
2. a commit to introduce `participant_type` to `getinfo` command

Sorry for the additional work :)

Reply:
Okay great, I'll start working on that then :D